configurationdemo/source/objc/ConfigurationDemo/ViewController.m

```diff
@@ -106,7 +106,7 @@ - (void)onConfigurationUpdate:(KAAConfigurationConfiguration *)configuration {
 
 - (void)displayConfiguration {
     KAAConfigurationConfiguration *configuration = [self.kaaClient getConfiguration];
-    [self addLogWithText:[NSString stringWithFormat:@"Sampling period is %d", configuration.samplePeriod]];
+    [self addLogWithText:[NSString stringWithFormat:@"Sampling period is %d sec.", configuration.samplePeriod]];
```

Comment:
Please folllow the rules used in the Bundle description. E.g.:

`[pool-2-thread-1] INFO  o.k.k.d.c.ConfigurationDemo - Sampling period is now set to 10 seconds`